bedrock/mozorg/templates/mozorg/about/governance/policies/reviewers.html

```diff
@@ -0,0 +1,147 @@
+    <dt>{{ _('All changes that affect how code modules interact') }}</dt>
+    <dd>{{ _('Any other changes that fall outside of the above rules, but affect how two or more code modules function, must have super-review.') }}</dd>
+  </dl>
+</section>
+
+<section id="exceptions">
+  <h2>{{ _('Exceptions') }}</h2>
+  <p>{{ _('<a href="%(ted)s">Ted Mielczarek</a> is the <a href="%(autoconf)s">autoconf</a> module owner for Unix, OS2, and other gmake-friendly platforms. His review is sufficient to get changes to those files checked in.')|format(ted='mailto:ted.mielczarek@gmail.com', autoconf='https://wiki.mozilla.org/Modules/Core#Build_Config') }}</p>
+  <p>{{ _('The <abbr title="Netscape Portable Runtime">NSPR</abbr> module is mature, stable and has been used in Mozilla antecedents for years. Peer and module owner review is sufficient to get those files checked in.') }}</p>
+  <p>{{ _('The <a href="%(module)s">JavaScript</a> engine is a self-contained area, with dependencies only on another self-contained area (<abbr title="Netscape Portable Runtime">NSPR</abbr>). Module owner review suffices to change <a href="%(source)s"><abbr title="JavaScript">JS</abbr></a> code.')|format(module='https://wiki.mozilla.org/Modules/Core#JavaScript', source='http://lxr.mozilla.org/mozilla/source/js/src/') }}</p>
+  <p>{{ _('Changes to module <a href="%s"><abbr title="Personal Security Manager">PSM</abbr></a> require at least one thorough owner/peer review. If the person requesting the change is neither the owner nor a peer, a second person from the group of {PSM owners, PSM peers, super-reviewers} must approve the change. Changes to UI that is shared amongst all Mozilla applications (e.g. error messages, password prompts, information windows, certificate manager) shall be reviewed/approved by the Mozilla Foundation’s User Experience team. Changes to UI in the primary application windows (e.g. Firefox preferences, security button in mail, status bar items) must be reviewed/approved by the respective application UI owners.')|format('https://developer.mozilla.org/docs/Mozilla/Projects/PSM') }}</p>
+  <p>{{ _('SeaMonkey specific changes (suite/ in comm-central) are covered by their <a href="%s">own review policy.</a>')|format('http://www.seamonkey-project.org/dev/review-and-flags') }}</p>
+  <p>{{ _('Calendar specific changes (calendar/ in comm-central) do not require superreview as detailed <a href="%s">by their policy.</a>')|format('https://wiki.mozilla.org/Calendar:Review_Process') }}</p>
+  <p>{{ _('In general, a well-owned module may make a request to <a href="%s">super-reviewers</a> to be exempt from super-review requirements.')|format('mailto:super-reviewers@mozilla.org') }}</p>
+  <p>{{ _('See <a href="%s">despot.cgi</a> for more on modules, <abbr title="also known as">a.k.a.</abbr> partitions.')|format('https://despot.mozilla.org/despot.cgi') }}</p>
```

Comment:
Link to https://despot.mozilla.org/despot.cgi is timing out for me? Doesn't need to block this PR, just making note here.